xpresso/_utils/media_type_validator.py

```diff
@@ -24,10 +24,7 @@ def matches(self, media_type: typing.Optional[str]) -> bool:
         if media_type is None:
             return False
         media_type = next(iter(media_type.split(";"))).lower()
-        for accepted in self.accepted:
-            if accepted.match(media_type):
-                return True
-        return False
+        return any(accepted.match(media_type) for accepted in self.accepted)
```

Comment:
I don't care much either way. I tend to prefer the more verbose version that doesn't use language specific features (remember that `any` is a builtin and that you're actually creating a generator in there), but I'd be okay replacing it.